grid/project-management/projects/profile.supr

```diff
@@ -7,8 +7,9 @@ version = "1.0.1"
 
 """
 List Projects
+List projects in project management systems.
 """
-usecase ListProjects {
+usecase ListProjects safe {
```

Comment:
Right, good eye - though I don't think that on merging, `1.1` won't get updated anyway, since the CI will only run for the latest `2.0` commit, right?

Reply:
Impact - use case safety is `unsafe` by default, in practice meaning the use case ends up being a GraphQL mutation, if used via one-service.

Reply:
> though I don't think that on merging, 1.1 won't get updated anyway, since the CI will only run for the latest 2.0 commit, right?

Yes CI will run the publish on latest commit only.

> Impact - use case safety is unsafe by default, in practice meaning the use case ends up being a GraphQL mutation, if used via one-service.

Then it is breaking change and major version should be bumped.

Reply:
Ok, I think this is resolved since we're bumping the major version anyway - the commits were made separately more so for reviewing clarity. Thanks!

Reply:
(Or should I rewrite the commits so that there isn't a "wrong" version in the repo's history?)

Reply:
Oh wait, this is **projects**. I'll bump the version :sweat_smile: